frontend/src/k8sTypes.ts

```diff
@@ -286,6 +287,15 @@ export type ProjectKind = K8sResourceCommon & {
   };
 };
 
+export type NamespaceKind = K8sResourceCommon & {
+  metadata: {
+    annotations?: DisplayNameAnnotations;
+  };
+  status?: {
+    phase: 'Active' | 'Terminating';
+  };
```

Comment:
@cam-garrison is this something we need?

Reply:
Added for use when fetching namespace annotation [here](https://github.com/opendatahub-io/odh-dashboard/pull/1088/files/dd22cd76e6aa2e2158829365ca209949c0549264#diff-05d6c60cadfff01f00f888ddbd48604ebd587d0a37f5678b713d653e19e4b973R22)

Reply:
No, just use `ProjectKind` instead, you get pretty much the same and openshift permissions are better to handle with projects

Reply:
+1 https://github.com/opendatahub-io/odh-dashboard/pull/1088/commits/8376c2818e44bd39b6f995b25aaa4df9bdacea5a 

removed in this commit for using ProjectKind